Sources/NIO/DatagramVectorReadManager.swift

```diff
@@ -113,7 +113,10 @@ struct DatagramVectorReadManager {
             assert(messagesProcessed == 0)
             return .none
         case .processed(let messagesProcessed):
-            return self.buildMessages(messageCount: messagesProcessed, sliceSize: messageSize, buffer: &buffer)
+            buffer.moveWriterIndex(to: messageSize * messagesProcessed)
```

Comment:
@Lukasa requesting re-review because of this change.

Reply:
basically need you to confirm that this is correct (test crashed on Linux otherwise)

Reply:
LGTM.